### PR TITLE
add a test to verify we can init sandcastle

### PR DESCRIPTION
**Pull request by TomasTomecek**
https://github.com/packit/packit-service/issues/908

### CODE REVIEW

**Review by jpopelka — APPROVED** [review]

File: tests/unit/test_sandcastle.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
```
Comment (lachmanfrantisek) [inline tests/unit/test_sandcastle.py:21]: We can use the shorter version.

**Review by lachmanfrantisek — APPROVED** [review]

**Comment by TomasTomecek** [comment]
oh, I need to move it to requre tests

File: tests/unit/test_sandcastle.py (hunk 1 above)
Comment (TomasTomecek) [reply tests/unit/test_sandcastle.py:21]: I was lazy to look for it :D

**Comment by TomasTomecek** [comment]
recheck

**Pull request merged**